autometa/taxonomy/majority_vote.py

```diff
@@ -230,7 +215,6 @@ def rank_taxids(ctg_lcas, ncbi_dir, verbose=False):
         top_taxids[contig] = acceptedTaxid
     return top_taxids
 
-
 def majority_vote(fasta, ncbi_dir, outdir, votes_fname, lca_fname=None, **kwargs):
     """Wrapper for modified majority voting algorithm from Autometa 1.0
 
```

Comment:
See first point in issue #42. More detail about the arguments would be nice, for example presumably `lca_fname` will be the filename written in `outdir`?

Reply:
Can probably copy/paste from the help messages in `main`.